utils/math/averager_heap_test.go

```diff
@@ -42,67 +45,69 @@ func TestAveragerHeap(t *testing.T) {
 	}
 
 	for _, test := range tests {
-		_, _, ok := test.h.Pop()
-		require.False(ok)
+		t.Run(test.name, func(t *testing.T) {
```

Comment:
Unrelated change I could unstage but this was bothering me when I was passing over the unit tests